brightener.h

```diff
@@ -6,7 +6,5 @@ class ImageBrightener {
 	std::shared_ptr<Image> m_inputImage;
 public:
 	ImageBrightener(std::shared_ptr<Image> inputImage);
-	bool ValidateImage();
```

Comment:
Good removal of unwanted functions. But is it needed to have the `BrightenWholeImage` function in a class with a data-member? Can we have it as a simple function accepting the image to be brightened, or is there a need for an object with state?